packages/server/graphql/mutations/createJiraTaskIntegration.ts

```diff
@@ -69,15 +67,41 @@ export default {
       )
     }
 
-    const viewerAuth = await dataLoader.get('freshAtlassianAuth').load({teamId, userId: viewerId})
-    if (!viewerAuth) {
-      return standardError(new Error('The assignee does not have access to Jira'), {
+    const [viewerAuth, assigneeAuth, team, teamMembers] = await Promise.all([
+      dataLoader.get('freshAtlassianAuth').load({teamId, userId: viewerId}),
+      userId && dataLoader.get('freshAtlassianAuth').load({teamId, userId}),
```

Comment:
ternary for strong types, a short circuit could result in the value of `false` which isn't a `PromiseLike` type


```ts
// from lib.d.ts
    then<TResult1 = T, TResult2 = never>(onfulfilled?: ((value: T) => TResult1 | PromiseLike<TResult1>) | undefined | null, onrejected?: ((reason: any) => TResult2 | PromiseLike<TResult2>) | undefined | null): PromiseLike<TResult1 | TResult2>;
```